packages/mongodb-chatbot-ui/src/Message.tsx

```diff
@@ -74,42 +74,23 @@ const styles = {
     display: flex;
     flex-direction: column;
 
-    & * {
-      line-height: 28px;
-    }
-
-    & p {
-      font-weight: 400;
-    }
-
-    & a {
-      font-weight: 400;
-    }
-
     & li {
       white-space: normal;
-      margin-top: -1.5rem;
-
-      & li {
-        margin-top: 0;
-      }
     }
 
-    & code {
-      white-space: pre-wrap;
+    & ol, & ul {
+      overflow-wrap: anywhere;
     }
-  `,
-  // End hacky fix
-  markdown_list: css`
-    overflow-wrap: anywhere;
-    margin-bottom: -1.5rem;
 
-    & ul {
-      margin-bottom: 0;
+    & h1, & h2, & h3 {
+      & +ol, & +ul {
+        margin-top: 0;
+        /* background-color: ${palette.blue.dark2}; */
```

Comment:
will this support both dark and light mode?

Reply:
Meant to delete this line! I used it to highlight the targeted elements in dev.